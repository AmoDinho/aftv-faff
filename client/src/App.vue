<template>
  <div id="app">
    <img alt="Vue logo" src="./assets/logo.png">
    <ApolloExample msg="Welcome to Your Vue.js + TypeScript App"/>
  </div>
</template>

<script lang="ts">
import Vue from 'vue';
import ApolloExample from './components/ApolloExample.vue';

export default Vue.extend({
  name: 'app',
  components: {
    ApolloExample,
  },
});
</script>

<style lang="scss">
#app {
  font-family: 'Avenir', Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  color: #2c3e50;
  margin-top: 60px;
}
</style>
