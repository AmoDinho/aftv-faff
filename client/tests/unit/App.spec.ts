import { createLocalVue, shallowMount, mount } from '@vue/test-utils'
import App from '../../src/App.vue'
describe('<App/>', () => {
    let localVue: any
    beforeEach(() => {
        localVue = createLocalVue()

    })


    test('Display all FanCams', () => {
        const wrapper = shallowMount(App, { localVue })
        wrapper.setData({
            getFanCams: [
                {
                    ID: '23434343434',
                    title: 'Wenger left us in a mess',
                    url: '/wertyirtrtid=23'
                },
                {
                    ID: '54545454545',
                    title: 'Chambers will be missed',
                    url: '/eererfid=23'
                },
                {
                    ID: '36567445',
                    title: 'The board are deliousional',
                    url: '/tgdfdfdfdid=23'
                }
            ]
        })
    })
})