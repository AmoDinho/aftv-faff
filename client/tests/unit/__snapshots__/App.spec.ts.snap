// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> Display all FanCams 1`] = `
<div
  id="app"
>
  <img
    alt="Vue logo"
    src="./assets/logo.png"
  />
   
  <home-stub />
</div>
`;
